sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/webflux/ReactorUtils.java

```diff
@@ -36,20 +39,20 @@ public static <T> Mono<T> withSentry(final @NotNull Mono<T> mono) {
    */
   @ApiStatus.Experimental
   public static <T> Mono<T> withSentryNewMainHubClone(final @NotNull Mono<T> mono) {
-    final @NotNull IHub hub = Sentry.cloneMainHub();
+    final @NotNull IScopes hub = Sentry.cloneMainHub();
```

Comment:
Rename to `scopes`

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366